os/fs/romfs/fs_romfs.c

```diff
@@ -656,6 +672,50 @@ static int romfs_dup(FAR const struct file *oldp, FAR struct file *newp)
 }
 
 /****************************************************************************
+ * Name: romfs_fstat
+ *
+ * Description:
+ *   Obtain information about an open file associated with the file
+ *   descriptor 'fd', and will write it to the area pointed to by 'buf'.
```

Comment:
you may have to change ,  file descriptor 'fd' => file pointer 'filep'

Reply:
Done.